### DISCUSSION TITLE
Porting old user IDs to new users

### DISCUSSION THREAD

**Discussion by Larront** [immich-app/immich#2381] [Q&A]
Hi all, 

I've had an issue where my config drive failed so I've had to re-init Immich. However, my old photo library exists. My old users were nuked, and when I make a new user it gives them a new user ID. Is there any way I can reassociate the old library with the new (recreated) user?

**Comment by Quadrubo** [comment]
I'd manually go into the database and change the user id manually if you don't have the option to restore a proper backup

**Reply by Birbodes** [reply]
Unfortunately I have the same problem. How exactly did you access the database?

**Comment by lyhnet** [comment]
I tried opening the bash console of immich postgres container. But running postgres inside it says that it cannot be modified by root.

**Comment by davidpan** [comment]
refer to this link : https://immich.app/docs/FAQ#how-can-i-move-all-data-photos-persons-albums-from-one-user-to-another

**Comment by jonasmusall** [comment]
for anyone looking for the answer linked above, it was removed as "unsupported sql shenanigans" in df31eb12147d22f291ff6122e29040f7a8925941. if you still want to try it (being aware that your database might break), read this version of the documentation page: https://github.com/immich-app/immich/blob/5ef2553bca3142ca9d9ad6364dabbe7564912592/docs/docs/FAQ.mdx#how-can-i-move-all-data-photos-persons-albums-libraries-from-one-user-to-another
